$MirOS: ports/net/mcabber/patches/patch-mcabberrc_example,v 1.6 2008/06/25 18:16:58 tg Exp $
--- mcabberrc.example.orig	Sun Oct  5 10:48:00 2008
+++ mcabberrc.example	Sun Oct  5 20:54:13 2008
@@ -1,4 +1,5 @@
-# Sample ~/.mcabber/mcabberrc file
+# $MirOS: ports/net/mcabber/patches/patch-mcabberrc_example,v 1.6 2008/06/25 18:16:58 tg Exp $
+# Sample ~/.etc/mcabber/mcabberrc file
 
 # Note about this file syntax:
 # - Leading and trailing spaces are ignored.
@@ -52,7 +53,7 @@ set ignore_self_presence = 1
 set ssl = 0
 #set ssl_verify = -1
 #set ssl_cafile = /usr/share/ssl/certs/ca-bundle.crt
-#set ssl_capath =
+set ssl_capath = /etc/ssl/certs
 #set ssl_ciphers =
 
 # PGP support
@@ -102,7 +103,7 @@ set ssl = 0
 # If your mcabber has been built with Aspell support, you can enable spell
 # checking with the following options (you can't change them once mcabber is
 # running).
-#set aspell_enable = 1
+set aspell_enable = 0
 #set aspell_lang = en_EN
 #set aspell_encoding = iso8859-1
 #set aspell_encoding = UTF-8
@@ -125,7 +126,7 @@ set cmdhistory_lines = 250
 # Typing notifications, Chat States, Events (JEP-22/85)
 # Set disable_chatstates to 1 if you don't want to use typing notifications.
 # Note: changing this option once mcabber is running has no effect.
-#set disable_chatstates = 0
+set disable_chatstates = 1
 
 # History logging
 # You can save the messages history: set logging = 1
@@ -133,15 +134,15 @@ set cmdhistory_lines = 250
 # (and status messages) to the log files (default: 0).
 # You can load (read) the messages history: set load_logs = 1
 # If you enable load_logs, you can use the 'max_history_age' setting below.
-# Default logging directory (logging_dir) is $HOME/.mcabber/histo/
+# Default logging directory (logging_dir) is $HOME/.etc/mcabber/histo/
 # Defaults for logging, load_logs are 0 (disabled)
 # Note: the logging directory must exist if you enable logging, mcabber
 #       will not create it.
 # Note: these options, except 'max_history_age' and 'max_history_blocks',
 # are used at startup time.
-#set logging = 1
+set logging = 0
 #set load_logs = 1
-#set logging_dir = ~/.mcabber/histo/
+#set logging_dir = ~/.etc/mcabber/histo/
 #set logging_ignore_status = 1
 
 # Set log_muc_conf to 1 to enable MUC chatrooms logging (default = 0)
@@ -149,7 +150,7 @@ set cmdhistory_lines = 250
 # Set load_muc_logs to 1 to read MUC chatrooms logs (default = 0).  These
 # logs will be displayed in the buffer window _before_ any history received
 # from the server.
-#set load_muc_logs = 0
+set load_muc_logs = 1
 
 # When load_logs (or load_muc_logs) is enabled, you can specify a maximum
 # number of history days to load into memory with max_history_age.
@@ -160,7 +161,7 @@ set cmdhistory_lines = 250
 # mcabber can store the list of unread messages in a state file,
 # so that the message flags are set back at next startup.
 # Note that 'logging' must be enabled for this feature to work.
-#set statefile = ~/.mcabber/mcabber.state
+#set statefile = ~/.etc/mcabber/mcabber.state
 
 # You can specify a maximum number of data blocks per buffer (1 block contains
 # about 8kB).  The default is 0 (unlimited).  If set, this value must be > 2.
@@ -175,7 +176,7 @@ set cmdhistory_lines = 250
 # queries (XEP-0012).  Set iq_last_disable_when_notavail to 1 if you want
 # to disable Last Activity reports only when the not-available status is set.
 # (Default is 0 for both options)
-#set iq_last_disable = 0
+set iq_last_disable = 1
 #set iq_last_disable_when_notavail = 1
 #
 # Set iq_hide_requests to 1 if you don't want received IQ requests to be
@@ -207,12 +208,12 @@ set cmdhistory_lines = 250
 #   $events_command STATUS X jabber@id      (new buddy status is X)
 #   $events_command UNREAD N                (number of unread buddy buffers)
 # See sample script in contrib/ directory.
-#set events_command = ~/.mcabber/eventcmd
+set events_command = ~/.etc/mcabber/eventcmd
 #
 #set events_ignore_active_window = 0
 #
 #set event_log_files = 0
-#set event_log_dir = ~/.mcabber/event_files
+set event_log_dir = ~/.etc/mcabber/event_files
 
 # If you set 'eventcmd_use_nickname' to 1, mcabber will pass the nickname
 # (if it is defined) to the event script instead of the JID (default: 0).
@@ -244,7 +245,7 @@ set cmdhistory_lines = 250
 # When FIFO  is configured, you can turn it off and on in real time with
 # the 'fifo_ignore' option (default: 0).  When set to 1, the FIFO input is
 # still read but it is discarded.
-#set fifo_name = ~/.mcabber/mcabber.fifo
+set fifo_name = ~/.etc/mcabber/mcabber.fifo
 #set fifo_hide_commands = 0
 #set fifo_ignore = 0
 
@@ -255,7 +256,7 @@ set cmdhistory_lines = 250
 #  lvl 2: debug logging (XML, etc.)
 # Default is level 0, no trace logging
 #set tracelog_level = 1
-#set tracelog_file = ~/.mcabber/mcabber.log
+#set tracelog_file = ~/.etc/mcabber/mcabber.log
 
 # Set the auto-away timeout, in seconds.  If set to a value >0,
 # mcabber will change your status to away if no real activity is detected
@@ -317,7 +318,7 @@ set message_autoaway = Auto-away (idle)
 # If you want mcabber to react faster after hitting Escape, you can lower
 # this value by setting the ESCDELAY environment variable or setting the
 # 'escdelay' option.
-set escdelay = 50
+set escdelay = 250
 
 # Colors
 # Colors are: black, red, green, yellow, blue, magenta, cyan, white
@@ -449,11 +450,12 @@ set escdelay = 50
 # Set otr to 1 to enable OTR support.
 # You cannot change this option once mcabber is running.
 #set otr = 0
+#otrpolicy default opportunistic
 #
 # OTR directory
 # You can specify the OTR directory with the otr_dir option
-# (default = ~/.mcabber/otr/).
-#set otr_dir = "~/.mcabber/otr/"
+# (default = ~/.etc/mcabber/otr/).
+set otr_dir = "~/.etc/mcabber/otr/"
 
 # Aliases
 alias me = say /me
@@ -476,6 +478,10 @@ bind 276 = roster toggle
 # Ctrl-Up/Ctrl-Down bound to /buffer up/down  (like Ctrl-p/Ctrl-n)
 bind 521 = buffer up
 bind 514 = buffer down
+# Ctrl-J toggles msay enable/send
+bind 10 = msay toggle
+# Ctrl-D works as Delete, like in mksh
+bind 4 = iline char_fdel
 
 # Sample alias to join mcabber channel with "/jmc"
 #alias jmc = room join mcabber@conf.lilotux.net
